doc/users/whats_new.rst

```diff
@@ -460,14 +497,71 @@ Example
     subplot2grid(shape, loc, rowspan=1, colspan=1, fig=myfig)
 
 
-Interpolation in fill_betweenx
-------------------------------
+Interpolation in ``fill_betweenx``
+----------------------------------
 
 The ``interpolate`` parameter now exists for the method
-:func:`~matplotlib.axes.Axes.fill_betweenx`. This allows a user to interpolate
-the data and fill the areas in the crossover points, similarly to
-:func:`~matplotlib.axes.Axes.fill_between`.
+:func:`~matplotlib.axes.Axes.fill_betweenx`. This allows a user to
+interpolate the data and fill the areas in the crossover points,
+similarly to :func:`~matplotlib.axes.Axes.fill_between`.
+
+
+New keyword argument ``sep`` for EngFormatter
+---------------------------------------------
+
+A new ``sep`` keyword argument has been added to
+:class:`~matplotlib.ticker.EngFormatter` and provides a means to
+define the string that will be used between the value and its
+unit. The default string is " ", which preserves the former
+behavior. Additionally, the separator is now present between the value
+and its unit even in the absence of SI prefix.  There was formerly a
+bug that was causing strings like "3.14V" to be returned instead of
```

Comment:
adding bq's here and below will probably help making the difference (extra space) more visible